examples/src/main/java/com/vesoft/nebula/examples/GraphClientExample.java

```diff
@@ -88,8 +94,7 @@ public static void main(String[] args) {
         NebulaPool pool = new NebulaPool();
         Session session;
         try {
-            NebulaPoolConfig nebulaPoolConfig = new NebulaPoolConfig();
-            nebulaPoolConfig.setMaxConnSize(100);
+            NebulaPoolConfig nebulaPoolConfig = NebulaPoolConfig.builder().maxConnSize(100).build();
```

Comment:
Yes as a consumer of the lib i thought it was fine since its a compile error, easy to fix and provides for immutablility.

Reply:
IMO the important parts for this PR is the `pom.xml` as it upgrades and applys a good standard for the project along w/ spotless which makes the formating easy.

The rest with the configuration is fine either way, but in general it is bad practice to have configuration objects that are mutable. If you can change the configuration while is running that needs to be a feature, but in general I find that it just creates more problems. Its been best practice to make configuration immutable so it can't change in the middle of running it.

Reply:
Yeah, the mvnw change is great! And the spotless is convenient for contributors. I'am glad to merge your code with mvnw and spotless. 
And for the configuration, I totally agree with the way to build a configuration, I use it in [connector](https://github.com/vesoft-inc/nebula-spark-connector/blob/10006081763c794264a63e06a5e9250727f79391/example/src/main/scala/com/vesoft/nebula/examples/connector/NebulaSparkReaderExample.scala#L43) too. Since it involves changes on the user side, it needs to be confirmed with `pd`.

Reply:
So could you please split the configuration part to another pr? Then we can merge the mvnw and spotless first.

Reply:
Thanks for your pr. But I suggest postponing to the next x version to avoid incompatibility.